### PR TITLE
[Auto Logout] Xamarin Forms in AutoFill flow (iOS)

### PR DESCRIPTION
**Pull request by vincentsalucci**

### CODE REVIEW

File: src/App/Pages/Accounts/LoginPage.xaml [hunk 1]
```diff
@@ -74,7 +74,8 @@
                 </Grid>
             </StackLayout>
             <StackLayout Padding="10, 0">
```
Comment (kspearrin) [inline src/App/Pages/Accounts/LoginPage.xaml:76]: i'd move the visibility binding to this stack layout

File: src/App/Pages/Accounts/LoginPageViewModel.cs [hunk 2]
```diff
@@ -69,6 +69,8 @@ public string MasterPassword
         public Action StartTwoFactorAction { get; set; }
         public Action LoggedInAction { get; set; }
 
+        public bool HideHintButton { get; set; }
```
Comment (kspearrin) [inline src/App/Pages/Accounts/LoginPageViewModel.cs:72]: needs databinding getters/settings

Comment (vincentsalucci) [reply src/App/Pages/Accounts/LoginPageViewModel.cs]: Updated and fixed.

File: src/App/Pages/Accounts/LoginPage.xaml (hunk 1 above)
Comment (vincentsalucci) [reply src/App/Pages/Accounts/LoginPage.xaml]: Moved.

**Comment by kspearrin** [comment]
Slack message pasted here:

> 
> This is Vince's PR, but I kinda commandeered it (edited) 
> 5:04
> @vincentsalucci Can you move it out of draft?
> 5:04
> not sure how i can become the owner
> 5:05
> I also merged it with master a while back and it brought in some changes from a different PR. not sure why github does that
> 5:05
> So ignore the changes to AppResources
> 5:06
> Basically I had to abstract all the dialog calls from login and 2fa pages, do some hacking to get styles working (re email change with Alexey), abstract other stuff related to login and 2fa like yubikey, close buttons, etc

File: src/iOS.Autofill/CredentialProviderViewController.cs [hunk 3]
```diff
@@ -1,21 +1,27 @@
...
 using Bit.App.Abstractions;
-using Bit.App.Resources;
 using Bit.Core.Abstractions;
 using Bit.Core.Utilities;
 using Bit.iOS.Autofill.Models;
 using Bit.iOS.Core.Utilities;
 using Foundation;
 using System;
 using System.Threading.Tasks;
+using Bit.App.Pages;
```
Comment (cscharf) [inline src/iOS.Autofill/CredentialProviderViewController.cs:10]: Do you generally keep `using` statements in alphabetical order?

File: src/iOS.Autofill/CredentialProviderViewController.cs [hunk 4]
```diff
@@ -286,5 +287,44 @@ private void InitAppIfNeeded()
...
+
+            var navigationPage = new NavigationPage(loginPage);
+            var loginController = navigationPage.CreateViewController();
+            loginController.ModalPresentationStyle = UIModalPresentationStyle.FullScreen;
+            PresentViewController(loginController, true, null);
+        }
+
+        private void LaunchTwoFactorFlow()
+        {
+
```
Comment (cscharf) [inline src/iOS.Autofill/CredentialProviderViewController.cs:313]: Extra line should be removed

File: src/iOS.Extension/LoadingViewController.cs [hunk 5]
```diff
@@ -409,5 +419,44 @@ private bool IsAuthed()
...
+
+            var navigationPage = new NavigationPage(loginPage);
+            var loginController = navigationPage.CreateViewController();
+            loginController.ModalPresentationStyle = UIModalPresentationStyle.FullScreen;
+            PresentViewController(loginController, true, null);
+        }
+
+        private void LaunchTwoFactorFlow()
+        {
+
```
Comment (cscharf) [inline src/iOS.Extension/LoadingViewController.cs:445]: Extra blank line here also

File: src/iOS.Autofill/CredentialProviderViewController.cs (hunk 3 above)
Comment (kspearrin) [reply src/iOS.Autofill/CredentialProviderViewController.cs:10]: I have not sorted alpha in the past, though it appears that happens by default sometimes.

**Pull request merged**